test/test_conversion.py

```diff
@@ -270,3 +270,8 @@ class HAVCMode(enum.Enum):
         self.assertEqual(bytes([2]), knxdclient.encode_value(HAVCMode.Standby, knxdclient.KNXDPT.ENUM8))
         self.assertEqual(bytes([2]), knxdclient.encode_value(2, knxdclient.KNXDPT.ENUM8))
         self.assertEqual(2, knxdclient.decode_value(bytes([2]), knxdclient.KNXDPT.ENUM8))
+
+    def test_dpt232_conversion(self) -> None:
+        self.assertEqual((0x12, 0x34, 0x56),
+                         knxdclient.decode_value(knxdclient.encode_value((0x12, 0x34, 0x56), knxdclient.KNXDPT.COLOUR_RGB),
+                                                 knxdclient.KNXDPT.COLOUR_RGB))
```

Comment:
Maybe add another line to the test here that actually checks the encoded data? Something like 
```python
self.assertEqual(bytes(0x12, 0x34, 0x56), knxdclient.encode_value((0x12, 0x34, 0x56))
```